spec/factories.rb

```diff
@@ -10,6 +10,6 @@
   factory :application_choice do
     provider_ucas_code { Faker::Alphanumeric.alphanumeric(number: 3).upcase }
     application_form
-    status { :application_complete }
+    status { ApplicationStateChange.valid_states.sample }
```

Comment:
don't we care about the status? should not this affect some of the test (in theory)?

Reply:
I think tests that care about it set it explicitly?

Reply:
Yes correct. Tests that don't care about status have a random one:

https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/325/commits/dbe32c7666a6ec1c35074db24d93e1e8705cd452